.dockerignore

```diff
@@ -128,6 +131,8 @@ blocks/**/.env
 # generated files
 *.gen.*
 *.tsbuildinfo
+*.pdb
```

Comment:
Similar to above, never seen it.

Reply:
`.pdb` is a Windows thing AFAIR, still in favor of not adding it to the ignore tho.

Reply:
I'll remove this, but IMO it is unlikely we would ever want it committed as part of source. More about `.pdb` files: https://en.wikipedia.org/wiki/Program_database

Reply:
Addressed in https://github.com/hashintel/hash/pull/2938/commits/211e32f469a1e5fe75e195250422ebd111c781af